browser/playlist/playlist_service_factory.cc

```diff
@@ -99,6 +101,30 @@ class PlaylistServiceDelegateImpl : public PlaylistService::Delegate {
                        weak_ptr_factory_.GetWeakPtr(), std::move(callback)));
   }
 
+  void EnabledStateChanged(bool enabled) override {
```

Comment:
When it's disabled, maybe we need to close playlist panel if it's opened?

Reply:
Yeah, you have a point. Where do you think should it be handled? Is it okay to add some code to `SidebarContainerView::OnItemRemoved` ?

Reply:
I think you can request playlist panel closing here?

Reply:
Ah, ok. it could be generally handled - `closing panel when its panel item is removed` 👍🏼

Reply:
Oh, it looks like index could be a problem, if we deactivate something in the callback. Let me try update the active index inside Model as you agreed that it can be common behavior.

Reply:
> I think you can request playlist panel closing here?

Actually I found that your suggestion is much easier to close the panel. 👍